lib/livebook/runtime/standalone_init.ex

```diff
@@ -124,31 +111,33 @@ defmodule Livebook.Runtime.StandaloneInit do
     loop.(loop)
   end
 
+  # Note Windows does not handle escaped quotes and newlines the same way as Unix,
+  # so the string cannot have constructs newlines nor strings. That's why we pass
+  # the parent node name as ARGV and write the code avoiding newlines.
+  @child_node_eval_string """
+  [parent_node] = System.argv();\
+  init_ref = make_ref();\
+  parent_process = {node(), String.to_atom(parent_node)};\
+  send(parent_process, {:node_started, init_ref, node(), self()});\
+  receive do {:node_initialized, ^init_ref} ->\
+    manager_ref = Process.monitor(Livebook.Runtime.ErlDist.Manager);\
+    receive do {:DOWN, ^manager_ref, :process, _object, _reason} -> :ok end;\
+  after 10_000 ->\
+    :timeout;\
+  end\
+  """
+
+  if @child_node_eval_string =~ "\n" do
+    raise "invalid @child_node_eval_string, newline found: #{inspect(@child_node_eval_string)}"
+  end
```

Comment:
Wouldn't it be cleaner to `String.replace("\n", ";")` on the formatted code instead?

Reply:
You can't have two `;` in a row. So if you have two newlines, it would fail. We could handle all of these cases, squeeze, and so on but I preferred to write the abomination above for now.